client-spark/common/src/main/java/org/apache/spark/shuffle/RssSparkShuffleUtils.java

```diff
@@ -190,4 +200,33 @@ public static int getRequiredShuffleServerNumber(SparkConf sparkConf) {
     int taskConcurrencyPerServer = sparkConf.get(RssSparkConfig.RSS_ESTIMATE_TASK_CONCURRENCY_PER_SERVER);
     return (int) Math.ceil(estimateTaskConcurrency * 1.0 / taskConcurrencyPerServer);
   }
+
+  /**
+   * Get current active {@link SparkContext}. It should be called inside Driver since we don't mean to create any
+   * new {@link SparkContext} here.
+   *
+   * Note: We could use "SparkContext.getActive()" instead of "SparkContext.getOrCreate()" if the "getActive" method
+   * is not declared as package private in Scala.
+   * @return Active SparkContext created by Driver.
+   */
+  public static SparkContext getActiveSparkContext() {
+    return SparkContext.getOrCreate();
+  }
+
+  /**
+   * create broadcast variable of {@link ShuffleHandleInfo}
+   *
+   * @param sc                    expose for easy unit-test
+   * @param shuffleId
+   * @param partitionToServers
+   * @param storageInfo
+   * @return Broadcast variable registered for auto cleanup
+   */
+  public static Broadcast<byte[]> createPartShuffleServerMap(SparkContext sc, int shuffleId,
+      Map<Integer, List<ShuffleServerInfo>> partitionToServers, RemoteStorageInfo storageInfo) {
+    ShuffleHandleInfo partServerMap = new ShuffleHandleInfo(shuffleId, partitionToServers, storageInfo);
```

Comment:
`partServerMap` should be changed  to a better name.

Reply:
Let me change it.

Reply:
This line has not been changed. There are other places with the same problem, please check.

Reply:
I changed it to broadcastShuffleHdlInfo. Please double-check.

Reply:
I mean this line:`ShuffleHandleInfo partServerMap = new ShuffleHandleInfo(shuffleId, partitionToServers, storageInfo);`
Not only this method name.

Reply:
ok, I get you.